.travis.yml

```diff
@@ -20,7 +20,7 @@ install:
   - source devtools/travis-ci/install.sh
   - export PYTHONUNBUFFERED=true
   # Add org channel
-  - conda config --add channels omnia --add channels conda-forge
+  - conda config --add channels omnia --add channels conda-forge --add channels salilab
```

Comment:
It looks like the `omnia` channel already has a `modeller` package. Not sure if it's the same.

Reply:
I'm not sure, but I *think* they are different